### PR TITLE
[TR] Guides

### PR DESCRIPTION
**Pull request by Lxran**
Added the Turkish translation of the "Guides" article.

### CODE REVIEW

File: wiki/Guides/tr.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# Kılavuzlar
+
+aşağıda osu!community üyeleri tarafından yaratılmış; pek çoğu osu!forumlarda başlayıp buraya taşınmış olan kılavuzların sıralı listesi mevcuttur.
```
Comment (zeusminus) [inline wiki/Guides/tr.md:3]:
```suggestion
Aşağıda osu!community üyeleri tarafından yaratılmış; pek çoğu osu!forumlarda başlayıp osu! wiki'ye taşınmış olan kılavuzların sıralı listesi mevcuttur.
```

File: wiki/Guides/tr.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+## Beatmap oluşturma
+
+*Ana sayfa: [Beatmap oluşturma](/wiki/Beatmapping)*
+
+- [Özel Vuruş Sesleri Ekleme](Adding_Custom_Hitsounds)
+- [Ses Düzenleme](Audio_Editing)
+- [Sanatçı ya da Başlık Adı Değiştirme](Changing_the_Artist_or_Title)
+- [Başlık Metni Değiştirme](/wiki/Beatmaps/Title_Text#changing-title-text)
```
Comment (zeusminus) [inline wiki/Guides/tr.md:12]:
```suggestion
- [Başlık Metnini Değiştirme](/wiki/Beatmaps/Title_Text#changing-title-text)
```

File: wiki/Guides/tr.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+- [Collab Bilgilendirme](Collab_Information)
```
Comment (zeusminus) [inline wiki/Guides/tr.md:13]:
```suggestion
- [Collab Bilgisi](Collab_Information)
```

File: wiki/Guides/tr.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+- [Mapinizi Modlatmak](Getting_Your_Map_Modded)
+
+## Oynama
+
+- [Yeni Başlayanlar İçin Öğretici](Beginner's_Tutorial)
+- [osu!mania Nasıl Oynanır](How_to_Play_osu!mania)
+- [Ofset Sihirbazı Nasıl Kullanılır](How_to_Use_the_Offset_Wizard)
+- [Beatmaplari Aramak ve İndirmek](Searching_and_Downloading_Beatmaps)
+- [Tablet Satın Almak](Tablet_Purchase)
+- [Yetenek Geliştirmek Hakkında İpuçları](Tips_and_Tricks_on_Skill_Improvement)
```
Comment (zeusminus) [inline wiki/Guides/tr.md:38]:
```suggestion
- [Yetenek Geliştirme Üzerine İpuçları](Tips_and_Tricks_on_Skill_Improvement)
```

File: wiki/Guides/tr.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+
+## Tema yapımı
+
+*Ana sayfa: [Tema Yapımı](/wiki/Skinning)*
```
Comment (zeusminus) [inline wiki/Guides/tr.md:42]:
```suggestion
*Ana sayfa: [Tema yapımı](/wiki/Skinning)*
```

File: wiki/Guides/tr.md [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+
+*Ayrıca bakınız: [Tema Yapımı Kılavuzu](/wiki/Skinning_Tutorial)*
```
Comment (zeusminus) [inline wiki/Guides/tr.md:44]:
```suggestion
*Ayrıca bakınız: [Tema Yapma Öğreticisi](/wiki/Skinning_Tutorial)*
```

File: wiki/Guides/tr.md [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+
+- [Karışık Arkaplanlar Kırpmak](Cropping_with_Complex_Backgrounds)
+- [Basit Arkaplanlar Kırpmak](Cropping_with_Simple_Backgrounds)
```
Comment (zeusminus) [inline wiki/Guides/tr.md:47]:
```suggestion
- [Karışık Arkaplanları Kırpmak](Cropping_with_Complex_Backgrounds)
- [Basit Arkaplanları Kırpmak](Cropping_with_Simple_Backgrounds)
```

File: wiki/Guides/tr.md [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+- [Düzgünce Ortalanmış Spinnerlar Yapmak](Making_Properly_Centered_Spinners)
+
+## Diğer
+
+- [Discord Rich Presence](Discord_Rich_Presence)
```
Comment (zeusminus) [inline wiki/Guides/tr.md:52]:
```suggestion
- [Discord Zengin İçerik](Discord_Rich_Presence)
```
let's use the Turkish term for it

Comment (Lxran) [reply wiki/Guides/tr.md:52]: I'm going to leave it as is for now as the official Turkish web page of Discord didn't bother translating it.

**Pull request merged**